plc4j/integrations/apache-kafka/docker-compose.yml

```diff
@@ -0,0 +1,101 @@
+version: '3'
```

Comment:
It looks like you've used a mix of confluent versions, latest, 6.0.0 and 6.0.1.

6.0.1, is also far from the latest (7.3.3)

It also might be nicer to have the PLC4X version as an environment variable so it is easier to find when changing it for a release.

I'm guessing you've grabbed a base docker-compose file from Confluent, can you confirm where you got it from so that we can check what license it was released under?